packages/genesys-spark-components/src/components/beta/gux-avatar/example.html

```diff
@@ -108,6 +108,16 @@ <h2>Dynamic Example</h2>
       </gux-dropdown>
       <label slot="label">Presence</label>
     </gux-form-field-dropdown>
+    <gux-form-field-dropdown>
+      <gux-dropdown value="" id="update-uc-integration">
+        <gux-listbox aria-label="Update Uc Inegration">
+          <gux-option value="zoom">Zoom</gux-option>
+          <gux-option value="team">Teams</gux-option>
+          <gux-option value="8x8">8x8</gux-option>
+        </gux-listbox>
+      </gux-dropdown>
+      <label slot="label">UcIntegration</label>
+    </gux-form-field-dropdown>
```

Comment:
The Teams option is not working.
Should there be a Slack option here?

Reply:
Fixed the teams option